src/sources/generator.rs

```diff
@@ -18,25 +18,106 @@ use tokio::time::{interval, Duration};
 #[derive(Clone, Debug, Default, Deserialize, Serialize)]
 #[serde(deny_unknown_fields)]
 pub struct GeneratorConfig {
-    #[serde(default)]
-    sequence: bool,
-    lines: Vec<String>,
     #[serde(default)]
     batch_interval: Option<f64>,
     #[serde(default = "usize::max_value")]
     count: usize,
+    format: OutputFormat,
```

Comment:
What do you think about adding `#[serde(flatten)]` here, and ensuring that none of these fields conflict with those in `OutputFormat` below (ref https://serde.rs/attr-flatten.html)? That might even allow this new config to be compatible with the old style, given the default you have below.